File: JAN-FEB PROGRAM IN PYTHON/4.03.py

a = int(input("Enter first number: "))
b = int(input("Enter second number: "))

print("1. Addition")
print("2. Subtraction")
print("3. Multiplication")
print("4. Division")

choice = int(input("Enter your choice: "))


if choice == 1:
    result = a + b
    print("Result =", result)

elif choice == 2:
    result = a - b
    print("Result =", result)

elif choice == 3:
    result = a * b
    print("Result =", result)

elif choice == 4:
    if b != 0:
        result = a / b
        print("Result =", result)
    else:
        print("Cannot divide by zero")

else:
    print("Invalid Choice")

File: JAN-FEB PROGRAM IN C/4.03.c
#include <stdio.h>
int main()
{
    int a, b, choice, result;
    scanf("%d %d",&a,&b);
    scanf("%d", &choice);
    switch(choice) {
        case 1:
        result = a+b;
        printf("Result = %d",result);
        break;

        case 2:
        result = a-b;
        printf("Result = %d",result);
        break;

        case 3:
        result = a*b;
        printf("Result = %d", result);
        break;

        case 4:
        result = a/b;
        printf("Result = %d", result);
    }
    return 0;
}